pkg/portal/service/admin_api.go

```diff
@@ -23,6 +28,12 @@ type AdminAPIService struct {
 	AuthzAdder     AuthzAdder
 }
 
+type AuthzAuditContext struct {
+	Triggerer   string `json:"triggerer"`
```

Comment:
We have seen different usage of Admin API here. Let's be specific.

Let's replace `triggerer` by `usage`.

When the Admin API is used by portal for internal purpose, like collaborators. Specify `{ "usage": "internal" }`.

When the Admin API is proxied for an authenticated collaborator, specify `{ "usage": "proxy", "actor_user_id": "xxx" }`

Make usage a type alias of string. Add constants.